elements/lisk-transactions/src/4_multisignature_transaction.ts

```diff
@@ -224,6 +210,22 @@ export class MultisignatureTransaction extends BaseTransaction {
 		return errors;
 	}
 
+	public validateFee(): TransactionError | undefined {
```

Comment:
I don't think this function is meant to be public, used only within the class.

Reply:
Protected methods can not be overridden and we need to handle the special case of multi-signatures that's why public seems the only possible solution in this case but if you alternatives please share!